src/config.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "elastic",
+    "type": "executer",
+    "imgUrl": "logo.png",
+    "execProgram": "node",
+    "main": "app.js",
+    "version": "1.0.0",
+    "description": "Integrate to Elastic NodeJS client",
+    "settings": [
+      {
+        "name": "connetionString",
+        "viewName": "Connection String",
+        "type": "string"
+      }
+    ],
+    "methods": [
+      {
+        "name": "INDEX_DATA",
+        "viewName": "Index Elastic",
```

Comment:
The elastic `index` method is to actually add a document.
* rename to "Insert Document"
* rename param to "Document Object"